UNSUPPORTED_VERSIONS.md

```diff
@@ -8,39 +8,39 @@ Before publishing issue, PRs to any version of plugin check, is this version sup
       </tr>
       <tr>
         <td style="text-align: center"><a href="https://github.com/Falcion/UNITADE.md/tree/1.0.0">1.0.0</a></td>
-        <td style="text-align: center">❎</td>
+        <td style="text-align: center" align="center">❎</td>
       </tr>
```

Comment:
As other table cells, fix aligning to center by CSS styling in the HTML code without markdown interference.